packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -128,56 +131,38 @@ function GlobalStylesRevisionsMenu() {
 				globalStyles?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0,
 		};
 	}, [] );
-	const [ canReset, onReset ] = useGlobalStylesReset();
 	const { goTo } = useNavigator();
 	const { setEditorCanvasContainerView } = unlock(
 		useDispatch( editSiteStore )
 	);
 	const loadRevisions = () => {
 		setIsListViewOpened( false );
-		goTo( '/revisions' );
-		setEditorCanvasContainerView( 'global-styles-revisions' );
+
+		if ( ! isRevisionsOpened ) {
+			goTo( '/revisions' );
+			setEditorCanvasContainerView( 'global-styles-revisions' );
+		} else {
+			goTo( '/' );
+			setEditorCanvasContainerView( undefined );
+		}
 	};
 	const hasRevisions = revisionsCount > 0;
+	const isRevisionsOpened = useSelect(
```

Comment:
Tiny nit (non-blocking!): since `loadRevisions` checks `isRevisionsOpened`, I'd usually put `const isRevisionsOpened` before `const loadRevisions`. That way, if in the future anyone were to do something like wrapping `loadRevisions` in a `useCallback`, then the dependency is defined before it is used.